src/style.css

```diff
@@ -1,3 +1,79 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap';
+
 body {
   margin: 0;
+  background-color: #e1e3e1;
+  font-family: Roboto, system-ui;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  background-color: #fff;
+  align-items: center;
+}
+
+.img {
+  height: 40px;
+  margin-left: 50px;
+  padding: 0;
+}
+
+nav {
+  width: auto;
+  margin-right: 50px;
+}
+
+.ul__section {
```

Comment:
The class name '.ul__section' suggests that it represents a 'ul' element, but according to the checklist, class names should represent the meaning of the content, not the tag names. Consider renaming it to be more semantic and content-oriented.